<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>380</width>
         <height>227</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1" alignment="Qt::AlignBottom">
         <widget class="ColumnsDescriptionView" name="columnsDescription" native="true"/>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="RowsDescriptionView" name="rowsDescription" native="true"/>
        </item>
        <item row="1" column="1">
         <widget class="DrawingAreaView" name="drawingArea" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTest_add">
    <property name="title">
     <string>Test add</string>
    </property>
    <addaction name="actionAdd_blocks"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFree_drawing"/>
    <addaction name="actionField_defining"/>
    <addaction name="actionHuman_solving"/>
    <addaction name="actionAuto_solving"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTest_add"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAdd_blocks">
   <property name="text">
    <string>Add blocks</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionFree_drawing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Free drawing</string>
   </property>
  </action>
  <action name="actionField_defining">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Field defining</string>
   </property>
  </action>
  <action name="actionHuman_solving">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Human solving</string>
   </property>
  </action>
  <action name="actionAuto_solving">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto solving</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingAreaView</class>
   <extends>QWidget</extends>
   <header>widgets/DrawingAreaView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColumnsDescriptionView</class>
   <extends>QWidget</extends>
   <header>widgets/ColumnsDescriptionView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RowsDescriptionView</class>
   <extends>QWidget</extends>
   <header>widgets/RowsDescriptionView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
